tapir/coop/migrations/0041_auto_20231221_1403.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.23 on 2023-12-21 13:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("coop", "0040_alter_draftuser_share_owner"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="shareowner",
+            name="create_account_reminder_email_sent",
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AlterField(
+            model_name="shareownership",
+            name="share_owner",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="share_ownerships",
+                to="coop.shareowner",
+            ),
+        ),
```

Comment:
Unrelated to this PR, but this showed up when I run the migration.

Reply:
I must have forgotten to commit that migration before, sorry!